### PR TITLE
Update rest-api-quiz.md

### PR DESCRIPTION
**Pull request by LuigiZambetti**
Organize numbers.

### CODE REVIEW

File: rest-api/rest-api-quiz.md [hunk 1]
```diff
@@ -223,26 +223,19 @@
...
 - [ ] how long it takes to start a new programming language
 - [ ] how long it takes to install your SDK
 - [ ] how long it takes to read your documentation
 
-#### Q35. What is Time to First Hello World?
-- [x] how long it takes for a developer to do something with your API
-- [ ] how long it takes to start a new programming language
-- [ ] how long it takes to install your SDK
-- [ ] how long it takes to read your documentation
-
```
Comment (Ebazhanov) [inline rest-api/rest-api-quiz.md:243]: Hello, @LuigiZambetti! Thanks for raising this PR looks good, but I think you accidentally removed the answers, please double-check and revert it back. Let me know if you need any help from me.

File: rest-api/rest-api-quiz.md [hunk 2]
```diff
@@ -223,26 +223,19 @@
...
-
-#### Q36. Which response header tells the client and intermediaries that the response is not to be cached anywhere?
```
Comment (Ebazhanov) [inline rest-api/rest-api-quiz.md:245]: And this questions, should be present!

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@LuigiZambetti left some comments

File: rest-api/rest-api-quiz.md (hunk 1 above)
Comment (LuigiZambetti) [reply rest-api/rest-api-quiz.md:243]: That question was doubled. I've just remove one of both.

Comment (Ebazhanov) [reply rest-api/rest-api-quiz.md:243]: ok, agree

```
#### Q35. What is Time to First Hello World?
- [x] how long it takes for a developer to do something with your API
- [ ] how long it takes to start a new programming language
- [ ] how long it takes to install your SDK
- [ ] how long it takes to read your documentation

#### Q35. What is Time to First Hello World?
- [x] how long it takes for a developer to do something with your API
- [ ] how long it takes to start a new programming language
- [ ] how long it takes to install your SDK
- [ ] how long it takes to read your documentation
```

File: rest-api/rest-api-quiz.md (hunk 2 above)
Comment (Ebazhanov) [reply rest-api/rest-api-quiz.md:245]: But this questions with answers should stay? From you PR I see that you removed question
```
#### Q36. Which response header tells the client and intermediaries that the response is not to be cached anywhere?
- [ ] Cache-State: none
- [ ] Expires:-1
- [ ] Cache-Control: no-cache
- [x] Cache-Control: no-store
```

**Review by Ebazhanov — APPROVED** [review]
@LuigiZambetti thanks all good! Great contribution! :raised_hands:

**Comment by Ebazhanov** [comment]
@all-contributors please add @LuigiZambetti for design, code

**Pull request merged**